python312Packages.apsw: 3.45.1.0 -> 3.45.2.0, python312Packages.appthreat-vulnerability-db: 5.6.6 -> 5.6.7,  dep-scan: 5.2.14 -> 5.3.2

fabaff opened the pull request:
## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

gador commented on pkgs/development/python-modules/apsw/default.nix:
```diff
@@ -10,7 +10,7 @@
 buildPythonPackage rec {
   pname = "apsw";
-  version = "3.45.1.0";
+  version = "3.45.2.0";
```
Wouldn't in make sense to merge this commit with the previous one?

fabaff commented:
Result of `nixpkgs-review pr 302035` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.nplusone</li>
    <li>python311Packages.nplusone.dist</li>
    <li>python312Packages.nplusone</li>
    <li>python312Packages.nplusone.dist</li>
  </ul>
</details>
<details>
  <summary>14 packages failed to build:</summary>
  <ul>
    <li>appthreat-depscan</li>
    <li>appthreat-depscan.dist</li>
    <li>dep-scan</li>
    <li>dep-scan.dist</li>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>python311Packages.flask-admin</li>
    <li>python311Packages.flask-admin.dist</li>
    <li>python312Packages.flask-admin</li>
    <li>python312Packages.flask-admin.dist</li>
    <li>s3ql</li>
    <li>s3ql.dist</li>
    <li>sublime-music</li>
    <li>sublime-music.dist</li>
  </ul>
</details>
<details>
  <summary>93 packages built:</summary>
  <ul>
    <li>activitywatch</li>
    <li>aw-qt</li>
    <li>aw-qt.dist</li>
    <li>aw-watcher-afk</li>
    <li>aw-watcher-afk.dist</li>
    <li>aw-watcher-window</li>
    <li>aw-watcher-window.dist</li>
    <li>calibre</li>
    <li>cozy</li>
    <li>etesync-dav</li>
    <li>etesync-dav.dist</li>
    <li>frigate</li>
    <li>gkraken</li>
    <li>gwe</li>
    <li>home-assistant-component-tests.matrix</li>
    <li>matrix-commander</li>
    <li>matrix-commander.dist</li>
    <li>opsdroid</li>
    <li>opsdroid.dist</li>
    <li>pantalaimon</li>
    <li>pantalaimon-headless</li>
    <li>pantalaimon-headless.dist</li>
    <li>pantalaimon.dist</li>
    <li>pgadmin4</li>
    <li>pgadmin4-desktopmode</li>
    <li>pgadmin4-desktopmode.dist</li>
    <li>pgadmin4.dist</li>
    <li>python311Packages.appthreat-vulnerability-db</li>
    <li>python311Packages.appthreat-vulnerability-db.dist</li>
    <li>python311Packages.apsw</li>
    <li>python311Packages.apsw.dist</li>
    <li>python311Packages.aw-client</li>
    <li>python311Packages.aw-client.dist</li>
    <li>python311Packages.aw-core</li>
    <li>python311Packages.aw-core.dist</li>
    <li>python311Packages.etesync</li>
    <li>python311Packages.etesync.dist</li>
    <li>python311Packages.flask-security-too</li>
    <li>python311Packages.flask-security-too.dist</li>
    <li>python311Packages.llama-index-vector-stores-postgres</li>
    <li>python311Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python311Packages.matrix-nio</li>
    <li>python311Packages.matrix-nio.dist</li>
    <li>python311Packages.peewee</li>
    <li>python311Packages.peewee-migrate</li>
    <li>python311Packages.peewee-migrate.dist</li>
    <li>python311Packages.peewee.dist</li>
    <li>python311Packages.pgvector</li>
    <li>python311Packages.pgvector.dist</li>
    <li>python311Packages.wtf-peewee</li>
    <li>python311Packages.wtf-peewee.dist</li>
    <li>python311Packages.yfinance</li>
    <li>python311Packages.yfinance.dist</li>
    <li>python311Packages.zulip</li>
    <li>python311Packages.zulip.dist</li>
    <li>python312Packages.appthreat-vulnerability-db</li>
    <li>python312Packages.appthreat-vulnerability-db.dist</li>
    <li>python312Packages.apsw</li>
    <li>python312Packages.apsw.dist</li>
    <li>python312Packages.aw-client</li>
    <li>python312Packages.aw-client.dist</li>
    <li>python312Packages.aw-core</li>
    <li>python312Packages.aw-core.dist</li>
    <li>python312Packages.etesync</li>
    <li>python312Packages.etesync.dist</li>
    <li>python312Packages.llama-index-vector-stores-postgres</li>
    <li>python312Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python312Packages.matrix-nio</li>
    <li>python312Packages.matrix-nio.dist</li>
    <li>python312Packages.peewee</li>
    <li>python312Packages.peewee-migrate</li>
    <li>python312Packages.peewee-migrate.dist</li>
    <li>python312Packages.peewee.dist</li>
    <li>python312Packages.pgvector</li>
    <li>python312Packages.pgvector.dist</li>
    <li>python312Packages.wtf-peewee</li>
    <li>python312Packages.wtf-peewee.dist</li>
    <li>python312Packages.yfinance</li>
    <li>python312Packages.yfinance.dist</li>
    <li>python312Packages.zulip</li>
    <li>python312Packages.zulip.dist</li>
    <li>semgrep</li>
    <li>semgrep.dist</li>
    <li>sqlite-web</li>
    <li>sqlite-web.dist</li>
    <li>unbook</li>
    <li>visidata</li>
    <li>visidata.dist</li>
    <li>vorta</li>
    <li>vorta.dist</li>
    <li>weechatScripts.weechat-matrix</li>
    <li>zulip-term</li>
    <li>zulip-term.dist</li>
  </ul>
</details>

fabaff commented:
Failures:

- s3ql: https://github.com/NixOS/nixpkgs/pull/303073

fabaff committed:
python312Packages.apsw: 3.45.1.0 -> 3.45.2.0

Diff: rogerbinns/apsw@refs/tags/3.45.1.0...3.45.2.0

Changelog: https://github.com/rogerbinns/apsw/releases/tag/3.45.2.0

fabaff committed:
python312Packages.appthreat-vulnerability-db: format with nixfmt

fabaff committed:
python312Packages.appthreat-vulnerability-db: 5.6.6 -> 5.6.7

Diff: https://github.com/AppThreat/vulnerability-db/compare/refs/tags/v5.6.6...5.6.7

Changelog: https://github.com/AppThreat/vulnerability-db/releases/tag/v5.6.7

fabaff committed:
dep-scan: 5.2.14 -> 5.3.2

Diff: https://github.com/owasp-dep-scan/dep-scan/compare/refs/tags/v5.2.14...v5.3.2

Changelog: https://github.com/owasp-dep-scan/dep-scan/releases/tag/v5.3.2

fabaff committed:
dep-scan: format with nixfmt

fabaff committed:
appthreat-depscan: rename to dep-scan

fabaff commented:
Result of `nixpkgs-review pr 302035` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages marked as broken and skipped:</summary>
  <ul>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>python311Packages.nplusone</li>
    <li>python311Packages.nplusone.dist</li>
    <li>python312Packages.nplusone</li>
    <li>python312Packages.nplusone.dist</li>
  </ul>
</details>
<details>
  <summary>103 packages built:</summary>
  <ul>
    <li>activitywatch</li>
    <li>aw-qt</li>
    <li>aw-qt.dist</li>
    <li>aw-watcher-afk</li>
    <li>aw-watcher-afk.dist</li>
    <li>aw-watcher-window</li>
    <li>aw-watcher-window.dist</li>
    <li>calibre</li>
    <li>cozy</li>
    <li>dep-scan</li>
    <li>dep-scan.dist</li>
    <li>etesync-dav</li>
    <li>etesync-dav.dist</li>
    <li>frigate</li>
    <li>gkraken</li>
    <li>gwe</li>
    <li>home-assistant-component-tests.matrix</li>
    <li>matrix-commander</li>
    <li>matrix-commander.dist</li>
    <li>opsdroid</li>
    <li>opsdroid.dist</li>
    <li>pantalaimon</li>
    <li>pantalaimon-headless</li>
    <li>pantalaimon-headless.dist</li>
    <li>pantalaimon.dist</li>
    <li>pgadmin4</li>
    <li>pgadmin4-desktopmode</li>
    <li>pgadmin4-desktopmode.dist</li>
    <li>pgadmin4.dist</li>
    <li>python311Packages.appthreat-vulnerability-db</li>
    <li>python311Packages.appthreat-vulnerability-db.dist</li>
    <li>python311Packages.apsw</li>
    <li>python311Packages.apsw.dist</li>
    <li>python311Packages.aw-client</li>
    <li>python311Packages.aw-client.dist</li>
    <li>python311Packages.aw-core</li>
    <li>python311Packages.aw-core.dist</li>
    <li>python311Packages.etesync</li>
    <li>python311Packages.etesync.dist</li>
    <li>python311Packages.flask-admin</li>
    <li>python311Packages.flask-admin.dist</li>
    <li>python311Packages.flask-security-too</li>
    <li>python311Packages.flask-security-too.dist</li>
    <li>python311Packages.llama-index-vector-stores-postgres</li>
    <li>python311Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python311Packages.matrix-nio</li>
    <li>python311Packages.matrix-nio.dist</li>
    <li>python311Packages.peewee</li>
    <li>python311Packages.peewee-migrate</li>
    <li>python311Packages.peewee-migrate.dist</li>
    <li>python311Packages.peewee.dist</li>
    <li>python311Packages.pgvector</li>
    <li>python311Packages.pgvector.dist</li>
    <li>python311Packages.wtf-peewee</li>
    <li>python311Packages.wtf-peewee.dist</li>
    <li>python311Packages.yfinance</li>
    <li>python311Packages.yfinance.dist</li>
    <li>python311Packages.zulip</li>
    <li>python311Packages.zulip.dist</li>
    <li>python312Packages.appthreat-vulnerability-db</li>
    <li>python312Packages.appthreat-vulnerability-db.dist</li>
    <li>python312Packages.apsw</li>
    <li>python312Packages.apsw.dist</li>
    <li>python312Packages.aw-client</li>
    <li>python312Packages.aw-client.dist</li>
    <li>python312Packages.aw-core</li>
    <li>python312Packages.aw-core.dist</li>
    <li>python312Packages.etesync</li>
    <li>python312Packages.etesync.dist</li>
    <li>python312Packages.flask-admin</li>
    <li>python312Packages.flask-admin.dist</li>
    <li>python312Packages.llama-index-vector-stores-postgres</li>
    <li>python312Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python312Packages.matrix-nio</li>
    <li>python312Packages.matrix-nio.dist</li>
    <li>python312Packages.peewee</li>
    <li>python312Packages.peewee-migrate</li>
    <li>python312Packages.peewee-migrate.dist</li>
    <li>python312Packages.peewee.dist</li>
    <li>python312Packages.pgvector</li>
    <li>python312Packages.pgvector.dist</li>
    <li>python312Packages.wtf-peewee</li>
    <li>python312Packages.wtf-peewee.dist</li>
    <li>python312Packages.yfinance</li>
    <li>python312Packages.yfinance.dist</li>
    <li>python312Packages.zulip</li>
    <li>python312Packages.zulip.dist</li>
    <li>s3ql</li>
    <li>s3ql.dist</li>
    <li>semgrep</li>
    <li>semgrep.dist</li>
    <li>sqlite-web</li>
    <li>sqlite-web.dist</li>
    <li>sublime-music</li>
    <li>sublime-music.dist</li>
    <li>unbook</li>
    <li>visidata</li>
    <li>visidata.dist</li>
    <li>vorta</li>
    <li>vorta.dist</li>
    <li>weechatScripts.weechat-matrix</li>
    <li>zulip-term</li>
    <li>zulip-term.dist</li>
  </ul>
</details>
